cpp/tests/utilities/column_utilities.cu

```diff
@@ -941,5 +941,80 @@ bool validate_host_masks(std::vector<bitmask_type> const& expected_mask,
                      });
 }
 
+template <typename T, std::enable_if_t<cudf::is_fixed_point<T>()>*>
+std::pair<thrust::host_vector<T>, std::vector<bitmask_type>> to_host(column_view c)
+{
+  using namespace numeric;
+  using Rep = typename T::rep;
+
+  auto host_rep_types = thrust::host_vector<Rep>(c.size());
+
+  CUDF_CUDA_TRY(
+    cudaMemcpy(host_rep_types.data(), c.begin<Rep>(), c.size() * sizeof(Rep), cudaMemcpyDefault));
```

Comment:
Can we use `cudf::detail::make_std_vector_sync` or some vector factory like that, rather than using raw `cudaMemcpy` calls? We're using that function below, so it seems reasonable to do the same here.

Reply:
This `cudaMemcpy` was not introduced here but merely moved from `.hpp` file.
All of the `to_host` overloads return `thrust::host_vector` objects and the vector factories return `std::vector`.
I think the right thing would be change all of these since there are more than this one `cudMemcpy` due to the `to_host` signature. I believe this would be out of scope for this PR and I can create follow on one to correct all the `to_host` functions and callers.

Reply:
Great. Let's do a follow-up PR.